### PR TITLE
[FLINK-6837][table]Fix a test case name error, a small error message bug, And improve some message info

### PR DESCRIPTION
**Pull request by sunjincheng121**
Fix a test case name error,  a small error message bug, And improve some message info
- [x] General
  - The pull request references the related JIRA issue ("[FLINK-6837][table]Fix a small error message bug, And improve some message info.")
  - The pull request addresses only one issue
  - Each commit in the PR has a meaningful commit message (including the JIRA id)

- [ ] Documentation
  - Documentation has been added for new functionality
  - Old documentation affected by the pull request has been updated
  - JavaDoc for public methods has been added

- [ ] Tests & Build
  - Functionality added by the pull request is covered by tests
  - `mvn clean verify` has been executed successfully locally or a Travis build has passed

### CODE REVIEW

**Comment by sunjincheng121** [comment]
In this PR. Fix a test case name error,  a small error message bug, And change error message from ` "xxx" + variable` to `s"xxx: '${variable}'`.

File: flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/stream/table/OverWindowITCase.scala [hunk 1]
```diff
@@ -147,7 +147,7 @@ class OverWindowITCase extends StreamingWithStateTestBase {
   }
 
   @Test
-  def testProcTimeBoundedPartitionedRangeOver(): Unit = {
+  def testProcTimeBoundedPartitionedRowsOver(): Unit = {
```
Comment (zentol) [inline flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/stream/table/OverWindowITCase.scala]: Someone else will have to verify this change, but the other changes look good to me.

Comment (sunjincheng121) [inline flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/stream/table/OverWindowITCase.scala]: Sure. Thanks for the reviewing. @zentol

Comment (wuchong) [inline flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/stream/table/OverWindowITCase.scala]: +1 to this change

**Comment by wuchong** [comment]
The error in Travias CI seems not be relative to this PR, it is filed in https://issues.apache.org/jira/browse/FLINK-6836. So I will merge this...

**Pull request closed without merging**